packages/react/ui/src/core/useDate/useDate.ts

```diff
@@ -88,8 +90,17 @@ export default function useDate() {
           computedDate = parseDate(date.$date, lang);
         }
 
+        let dayjsFormat = "L";
```

Comment:
```suggestion
let dayjsFormat;
switch(format) {
// Apply one of our internal format
case "short" : dayjsFormat = "L"; break;
case "long" : dayjsFormat = "LL"; break;
case "abbr" : dayjsFormat = "ll"; break;
// Otherwise, expect a dayjs native format
default: dayjsFormat = format;
}
```